### PR TITLE
SELFDEV-370: [update] Replace links

### PR DESCRIPTION
**Pull request by slsriehl**
# [SELFDEV-370]

## What changed?
Provide a bulleted list in the present tense
* Replace links

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping @becomevocal @bc-stephen-hilliard 

[SELFDEV-370]: https://bigcommercecloud.atlassian.net/browse/SELFDEV-370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -628,7 +628,7 @@ paths:
 
         3. Supply a custom `tracking_link`: By providing a value for the `tracking_link` property, you can use your own tracking link within the BigCommerce control panel and in customer-facing emails. The API response will return your supplied tracking link as part of the `tracking_link` property in the response. In situations when there isn't a `generated_tracking_link`, the property in the API response will remain empty. 
         
-        Acceptable values for `shipping_provider` include an empty string (`""`), `auspost`, `carrier_{your_carrier_id}` (only used if the carrier is a [third-party Shipping Provider](/api-docs/providers/shipping)), `canadapost`, `endicia`, `usps`, `fedex`, `royalmail`, `ups`, `upsready`, `upsonline`, or `shipperhq`.
+        Acceptable values for `shipping_provider` include an empty string (`""`), `auspost`, `carrier_{your_carrier_id}` (only used if the carrier is a [third-party Shipping Provider](/docs/integrations/shipping)), `canadapost`, `endicia`, `usps`, `fedex`, `royalmail`, `ups`, `upsready`, `upsonline`, or `shipperhq`.
```
Comment (bc-andreadao) [inline reference/orders.v2.oas2.yml:631]:
```suggestion
        Acceptable values for `shipping_provider` include an empty string (`""`), `auspost`, `carrier_{your_carrier_id}` (only used if the carrier is a [third-party Shipping Provider](/docs/integrations/shipping), `canadapost`, `endicia`, `usps`, `fedex`, `royalmail`, `ups`, `upsready`, `upsonline`, or `shipperhq`.
```

File: reference/widgets.v3.yml [hunk 2]
```diff
@@ -1754,7 +1754,7 @@ components:
                 items:
                   type: object
                   title: widgetSchemaSetting_Base
-                  description: 'For examples of each type of setting, see [Page Builder > Schema Settings](/stencil-docs/page-builder/schema-settings/alignment) in Theme Docs.'
+                  description: 'For examples of each type of setting, see [Page Builder > Schema Settings](/stencil-docs/page-builder/schema-settings#alignment) in Theme Docs.'
```
Comment (bc-traciporter) [inline reference/widgets.v3.yml:1757]: This link doesn't work.

**Review by bc-traciporter — COMMENTED** [review]
I had a problem with one link.

File: reference/widgets.v3.yml (hunk 2 above)
Comment (bc-andreadao) [inline reference/widgets.v3.yml:1757]: This page was not found in the preview

**Review by bc-andreadao — CHANGES_REQUESTED** [review]
Requesting changes (see my above comment)

File: reference/widgets.v3.yml (hunk 2 above)
Comment (bc-andreadao) [reply reference/widgets.v3.yml:1757]: Agree with Traci, Found the same problem

**Review by bc-traciporter — DISMISSED** [review]
LGTM

**Review by bc-traciporter — APPROVED** [review]

File: reference/widgets.v3.yml (hunk 2 above)
Comment (slsriehl) [reply reference/widgets.v3.yml]: fixed

**Pull request merged**